tests/gui/test_widgets.py

```diff
@@ -1210,12 +1210,13 @@ def test_FileWidget_on_file_download_updates_items_when_uuid_matches(mocker, hom
     """
     mock_controller = mocker.MagicMock()
     mock_signal = mocker.MagicMock()  # not important for this test
-    file_ = factory.File(is_downloaded=True)
+    file_ = factory.File()
 
     session_maker = mocker.MagicMock()
     fw = FileWidget(session_maker, file_, mock_controller, mock_signal)
     fw.clear = mocker.MagicMock()
     fw.update = mocker.MagicMock()
+    fw.file_uuid = file_.uuid
```

Comment:
Is there a reason we have to set this manually? Wouldn't this be picked up by passing the `file_` object in?

Reply:
Resolving now that this is outdated after removing session code from gui.